package.json

```diff
@@ -68,13 +68,15 @@
   "dependencies": {
     "async": "~0.9.0",
     "compression": "^1.0.9",
-    "errorhandler": "^1.1.1",
-    "loopback": "^2.0.0",
-    "loopback-boot": "^2.0.0",
+    "cors": "^2.7.1",
+    "errorhandler": "^1.4.2",
```

Comment:
I believe `errorhandler` is not needed, since we are using `loopback#errorHandler` instead.